constants/talismans.js

```diff
@@ -183,7 +192,8 @@ module.exports = {
         SHADY_RING: { name: 'Shady Ring', rarity: 'uncommon' },
         CROOKED_ARTIFACT: { name: 'Crooked Artifact', rarity: 'rare' },
         SEAL_OF_THE_FAMILY: { name: 'Seal of the Family', rarity: 'epic' },
-        PARTY_HAT_CRAB: { name: 'Crab Hat of Celebration', rarity: 'special' },
+        PARTY_HAT_CRAB: { name: 'Crab Hat of Celebration - 2020', rarity: 'special' },
+        PARTY_HAT_CRAB_ANIMATED: { name: 'Crab Hat of Celebration - 2022', rarity: 'special' },
```

Comment:
Does this work with the current API? Thought there was a reason I didn't add it. Check the requests and make sure missing accessories or whatever isn't broken from it. Idk